Multisig/Presentation/UI Library/Colors.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Colors.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 14.04.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+extension Color {
```

Comment:
no code-gen tools this time? :)

Reply:
These are few, manual is fine.

Reply:
Also, I'd try to make process as simple as possible by using the standard set of tools  :)